fedimint-testing/Cargo.toml

```diff
@@ -37,6 +37,7 @@ ln-gateway = { version = "0.3.0-alpha", package = "fedimint-ln-gateway", path =
 ldk-node = { package = "fedimint-ldk-node", version = "0.1.0" }
 futures = "0.3"
 lightning-invoice = "0.26.0"
+lightning-invoice29 = { package = "lightning-invoice", version = "0.29.0" }
```

Comment:
ldk-node re-exports lightning invoice. Can probably just use that

Reply:
Requires https://github.com/fedimint/fedimint/pull/4354 since we're currently using [`fedimint-ldk-node` v0.1.0](https://crates.io/crates/fedimint-ldk-node) which re-exports `lightning-invoice` v0.23.0 instead of v0.29.0. Leaving this open until that PR is merged.